### PR TITLE
Modified the PR template to include README.md update reqirement if necessary

### LINKED ISSUE #32658: [Issue] Modified the PR template to include README.md update reqirement if necessary
This issue is automatically created based on existing pull request: magento/magento2#32610: Modified the PR template to include README.md update reqirement if necessary

---------
### Description (*)
With the introduction of the [README.md template](https://github.com/magento/devdocs/wiki/Magento-module-README.md), there is a requirement of describing module installation, structure and extension points in the readme.
We need to pay special attention to README.md always being up to date, because an outdated readme can do more harm than good. This is why all code contributors need to be aware that they are supposed to modify README.md as well when they create modifications that affect any of readme predefined sections.

### Related Pull Requests
-

### Fixed Issues (if relevant)
-

### Manual testing scenarios (*)
1. Attempt to create a Pull Request to this or any other github repository
2. When on the PR edit page, copy-paste the modified template content into your PR description area
3. Click Preview
4. Along the Contribution checklist checkboxes you should see a new checkbox with a description saying that README.md needs to be modified if necessary
5. Well, check if it looks beautiful :)  
6. Don't create that PR as you don't need to, unless you want to check that the markdown looks the same on an actual PR as in the Preview window

### Questions or comments
-

### Contribution checklist (*) - **THIS SECTION CONTAINS THE CHECKBOX THAT I ADDED**
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by bgorski**
### Description (*)
With the introduction of the [README.md template](https://github.com/magento/devdocs/wiki/Magento-module-README.md), there is a requirement of describing module installation, structure and extension points in the readme.
We need to pay special attention to README.md always being up to date, because an outdated readme can do more harm than good. This is why all code contributors need to be aware that they are supposed to modify README.md as well when they create modifications that affect any of readme predefined sections.

### Related Pull Requests
-

### Fixed Issues (if relevant)
-

### Manual testing scenarios (*)
1. Attempt to create a Pull Request to this or any other github repository
2. When on the PR edit page, copy-paste the modified template content into your PR description area
3. Click Preview
4. Along the Contribution checklist checkboxes you should see a new checkbox with a description saying that README.md needs to be modified if necessary
5. Well, check if it looks beautiful :)  
6. Don't create that PR as you don't need to, unless you want to check that the markdown looks the same on an actual PR as in the Preview window

### Questions or comments
-

### Contribution checklist (*) - **THIS SECTION CONTAINS THE CHECKBOX THAT I ADDED**
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#32658: Modified the PR template to include README.md update reqirement if necessary

### CODE REVIEW

**Comment by bgorski** [comment]
@magento run all tests

**Comment by bgorski** [comment]
@magento run WebAPI Tests

**Comment by sidolov** [comment]
Hi @dobooth ! Could you please help us with the review? Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -43,4 +43,5 @@
  - [ ] Pull request has a meaningful description of its purpose
  - [ ] All commits are accompanied by meaningful commit messages
  - [ ] All new or changed code is covered with unit/integration tests (if applicable)
+ - [ ] README.md files for modified modules are updated and included in the pull request if [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
```
Comment (dobooth) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
```

**Review by dobooth — APPROVED** [review]

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-8981](https://jira.corp.magento.com/browse/ENGCOM-8981) has been created to process this Pull Request

**Comment by engcom-Oscar** [comment]
@magento create issue

**Comment by engcom-Foxtrot** [comment]
@magento run all tests

**Pull request merged**